Makefile

```diff
@@ -360,6 +360,15 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
+.PHONY: prepare-release
+prepare-release: ## Prepare the manifests for OLM and Helm Chart for a release.
+	$(MAKE) bundle VERSION=$(VERSION) \
+		LIMITADOR_VERSION=$(LIMITADOR_VERSION) \
+		REPLACES_VERSION=$(REPLACES_VERSION) \
+		CHANNELS=$(CHANNELS)
```

Comment:
I think this `CHANNELS` set can be omitted? If you run `prepare-release` without setting `CHANNEL` it instead sets it to empty string, which overrides the equal check here and passes empty string https://github.com/Kuadrant/limitador-operator/blob/main/Makefile#L16-L18. Makes bundle validation fail

Reply:
Even if you remove and run like this all works `make prepare-release CHANNELS=preview`